From ffdeeb4e239963c58a5e23397f315588a56359c3 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Thu, 4 Apr 2019 14:57:00 +0800
Subject: [PATCH 17/30] extcon: ptn5150: fix compile error caused by implicit
 declaration of functions

Because of upstream commit 176aa3601213 ("extcon: Split out extcon header
file for consumer and provider device"), the declaration of some functions
have been moved to extcon-provide.h.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/extcon/extcon-ptn5150.c | 2 +-
 drivers/staging/typec/tcpci.c   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/extcon/extcon-ptn5150.c b/drivers/extcon/extcon-ptn5150.c
index 778b427699c0..8c9a9cb8b80e 100644
--- a/drivers/extcon/extcon-ptn5150.c
+++ b/drivers/extcon/extcon-ptn5150.c
@@ -23,7 +23,7 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/regmap.h>
-#include <linux/extcon.h>
+#include <linux/extcon-provider.h>
 #include <linux/gpio/consumer.h>
 
 /* PTN5150_REG_INT_STATUS */
diff --git a/drivers/staging/typec/tcpci.c b/drivers/staging/typec/tcpci.c
index 6a3a3d73f019..dbdb318d557a 100644
--- a/drivers/staging/typec/tcpci.c
+++ b/drivers/staging/typec/tcpci.c
@@ -15,7 +15,7 @@
 #include <linux/usb/tcpm.h>
 #include <linux/usb/typec.h>
 #include <linux/of_gpio.h>
-#include <linux/extcon.h>
+#include <linux/extcon-provider.h>
 
 #include "tcpci.h"
 
-- 
2.17.1

